webapp/snapcraft/logic.py

```diff
@@ -0,0 +1,36 @@
+from webapp import helpers
+from datetime import datetime, timedelta
```

Comment:
```python
from datetime import datetime, timedelta

from webapp import helpers
```

Reply:
what standard is this ordering and spacing? would be good to add some tooling the enforces it.

Reply:
yeah, my configuration does that for me automatically. I think you can find tools like `autoflake8` that you should be able to add.

`black` doesn't sort for you the imports which is a bit annoying.

Reply:
google search results for `autoflake8` returns nothing. Would be good to add something to the README or something based on your setup. It'd be easier if I could add a plugin to my editor.

Reply:
sorry it's `autoflake` to remove unused imports https://github.com/myint/autoflake
and `isort` to sort https://pypi.org/project/isort/